tensorflow/stream_executor/cuda/cuda_dnn.cc

```diff
@@ -1539,6 +1558,7 @@ class CudnnRnnSequenceTensorDescriptor
   TensorDescriptor handle_;
   RNNDataDescriptor rnn_data_handle_;
   std::vector<cudnnTensorDescriptor_t> handles_;  // Copies of handle_.
+  std::vector<uint8> seq_lengths_array_;
```

Comment:
The reason why we use uint8 rather int is because on the device side, we reuse the workspace allocator which only supports AllocateBytes and gives us DeviceMemory<uint8>. So, to call the ThenMemcpyH2D<uint8>(), we need to make sure the src and the dst have the same dtype.

I added some comment to explain this before the cast.